src/hyp3_sdk/hyp3.py

```diff
@@ -30,7 +30,8 @@ def __init__(
         api_url: str = PROD_API,
         username: str | None = None,
         password: str | None = None,
-        prompt: bool = False,
+        token: str | None = None,
+        prompt: str | bool | None = None,
```

Comment:
```suggestion
        prompt: Literal['password' , 'token'] | bool | None = None,
```
Will also need
```python
from typing import Literal
```

Reply:
Looks like Literal is already imported :) So I'll commit. Thanks, @jhkennedy